packages/components/src/templates/tab-group.stories.ts

```diff
@@ -0,0 +1,170 @@
+          justify-content: center;
+        }
+      </style>
+      <sd-tab-group activation id="centered">
+        <sd-tab slot="nav" panel="tab-1">Gender</sd-tab>
+        <sd-tab-panel name="tab-1">
+          <figure class="sd-media pb-4">
+            <img
+              src="./placeholders/images/collaboration.jpg"
+              alt="Two men in business attire sitting on a sofa in a modern office, smiling and looking at a laptop screen."
+              class="aspect-video object-cover"
+            />
+          </figure>
+          <p class="sd-paragraph">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie
+            phasellus dui vel id. Velit in sed non orci pellentesque vivamus nunc. At non tortor, sit neque tristique.
+            Facilisis commodo integer hendrerit tortor.
+          </p>
+        </sd-tab-panel>
+        <sd-tab slot="nav" panel="tab-2">Age diversity</sd-tab>
+        <sd-tab-panel name="tab-2">
+          <figure class="sd-media pb-4">
+            <img
+              src="./placeholders/images/family.jpg"
+              alt="A man with glasses and a beard holding two young children in his arms, smiling outdoors with trees in the background."
+              class="aspect-video object-cover"
+            />
+          </figure>
+          <p class="sd-paragraph">
+            Vivamus mattis mauris nec vulputate facilisis. Nulla facilisi. Pellentesque consectetur mi eget tristique
+            vestibulum. Nunc porta urna sit amet magna congue feugiat. Quisque commodo arcu ex, eget mollis magna porta
+            eu. Aliquam at turpis dolor. Quisque ut neque laoreet, cursus tellus non, venenatis elit.
+          </p>
+        </sd-tab-panel>
+        <sd-tab slot="nav" panel="tab-3">Education</sd-tab>
+        <sd-tab-panel name="tab-3">
+          <figure class="sd-media pb-4">
+            <img
+              src="./placeholders/images/workspace.jpg"
+              alt="Close-up of a pair of glasses, a pen, and an open notebook with notes written on it, on a desk next to a laptop."
+              class="aspect-video object-cover"
+            />
+          </figure>
+          <p class="sd-paragraph">
+            Sed dignissim ipsum vel ultricies interdum. Quisque posuere felis sit amet fringilla tristique. Donec velit
+            ex, viverra id lorem hendrerit, congue gravida justo. Fusce ultricies tellus arcu, nec suscipit metus
+            sagittis sed. Vestibulum lacinia enim eu tellus auctor, quis pulvinar felis varius.
+          </p>
+        </sd-tab-panel>
+        <sd-tab slot="nav" panel="tab-4">Background</sd-tab>
+        <sd-tab-panel name="tab-4">
+          <figure class="sd-media pb-4">
+            <img
+              src="./placeholders/images/collaboration.jpg"
+              alt="Two men in business attire sitting on a sofa in a modern office, smiling and looking at a laptop screen."
```

Comment:
@coraliefeil @MartaPintoTeixeira Do we omit copyrights here on purpose? Just thinking of best practices...

Reply:
we can have them (since is a template). updated in figma.

Reply:
I dunno. :D I think they are from Unsplash. Was just a proposal, don't want to overcomplicate things and to ensure, that we had that in mind.

Reply:
I like your solution @auroraVasconcelos

![CleanShot 2024-10-18 at 12 40 21@2x](https://github.com/user-attachments/assets/f9468dcb-3d07-4289-a40d-8b9a06252070)

Reply:
@mariohamann that is not actually a solution, it is just the example from the copyright. I believe that we could add there the real copyrights of the images.

Reply:
@mariohamann in a meeting with @MartaPintoTeixeira we agreed to simplify this issue by removing the copyright from this example. However, we believe this is a question the team should have clarified, since in the future we will certainly need to use copyrights with images other than the generic.jpg. It would be easier and faster to have an agreement on how to proceed in such cases.

Reply:
Discussed with the team, and updated with the correct copyrights @mariohamann and @MartaPintoTeixeira